governance/polls/Proposal - Rates V2 Poll - October 12, 2020.md

```diff
@@ -0,0 +1,72 @@
+* Add market competition-based rate setting methodology.
+* Minimize governance overhead.
+* Continual transparency of how rates are set.
+* Governance provides higher-level, plain-English business directives such as “set the stability fees on stablecoins to 0%” or “we want to focus on growth over profit”. These directives will follow the signal request format until a more formal MIP is created.
+
+Plan:
+
+* Scrap the current Stability Fee formula involving the Base Rate. The Base Rate will no longer be voted on after this proposal passes, and a MIP18 vote will be added to deprecate it formally.
+* Formation of the Rates Working Group. A somewhat loosely defined group consisting of Domain Team Facilitators and community members. Initial members being @hexonaut, @Primoz, @LongForWisdom, @monet-supply, @SebVentures and @Akiva.
+* Introduction of the Competitive Rate: A rate which is based on the competitive landscape. For example if our competition is offering borrowing at 5% for a particular asset, we may decide to offer that same asset for 4%.
+* The Rates Working Group will keep track of governance directives, risk premiums and competitive rates to produce a set of stability fees for each vault type. Mostly this will involve setting the rates by competitive analysis coming from risk.
+* The Risk Team will start producing Risk Premium / Competitive Rates on a monthly basis which get handed off to the Rates Working Group who will propose the Stability Fee rate changes.
+* Dai Savings Rate will remain at 0% until governance signals otherwise.
+* Development of a more formal MIP as this process is smoothed out.
+
+Responsibilities for the Rate Setting Group
+
+* Weekly meeting to determine adjustments.
+* Produce a rate adjustment proposal based on input from governance (in the form of signals or feedback), the Risk Team (Risk analysis), and from observation of the wider DeFi environment.
+* Forum post whenever adjustments are proposed including rationale and inviting discussion. Forum posts will be made 1 week in advance of the on-chain poll unless the changes are considered an “emergency”.
+* Bundled on-chain poll containing any proposed rate-changes for the week.
+* MIP to be produced within 6 months, group ceases to operate if MIP is not passed within 6 months. Deadline is April 1st, 2021.
+
+## Next Steps
+
+Upon a successful conclusion of this poll, the Base Rate vote will be halted for the week of October 19th. The Risk Domain Team to begin producing Risk Premium / Competitive Rates starting from the week of October 19th. It may take a bit to ramp up to all assets, but the team will do their best. The Rates Working Group would then propose an initial set of changes which will be voted on October 26th and added to the October 30th executive. After that we will move onto a monthly cycle of rate setting as outlined in the plan with the possibility of skipping November due to the closeness of the October vote.
```

Comment:
I thought we planned on doing changes on a weekly basis (if there are changes that should be made on that timescale)

Reply:
I'd also maybe format these as bullets. Feels like it might be easier to absorb. Something along the lines of:
* The plan described above will be implemented.
* There will be no base rate votes after the week of x.
* The Risk team will produce numbers starting week x + 1.
* The Rates group will  make their first proposal on week x + 1

Reply:
I was referring to Risk's aim to have monthly Risk Premium / Competitive Rate adjustments, but I guess that is a little unclear.